### PR TITLE
Layout_moyo_header

### PR DESCRIPTION
**Pull request by Hovera-Liubomyr**
https://Hovera-Liubomyr.github.io/layout_moyo-header/
https://Hovera-Liubomyr.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

**Review by danylo-onishchuk — CHANGES_REQUESTED** [review]
the link with blue color and line below is an active link. It should have class="is-active" and relevant styles.

need blue line

File: src/style.css [hunk 1]
```diff
@@ -1 +1,64 @@
...
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
```
Comment (aliceblunt8) [inline src/style.css:11]:
```suggestion
  padding: 0 50px;
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,64 @@
...
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  cursor: pointer;
+  padding: 23px 2px;
```
Comment (aliceblunt8) [inline src/style.css:32]: add height to link or nav menu, not paddings

File: src/style.css [hunk 3]
```diff
@@ -1 +1,64 @@
...
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  padding: 23px 1px;
```
Comment (aliceblunt8) [inline src/style.css:40]: remove, add height in px

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**